modules/installation-and-upgrade/pages/container-deployment/suma/server-migration-suma.adoc

```diff
@@ -82,6 +82,13 @@ By matching or exceeding these specifications, you can help ensure a smooth and
 mgradm migrate podman <source.fqdn>
 ----
 
+[IMPORTANT]
+====
+
+After successfully running the [command]``mgradm migrate` command, the Salt setup on all minions will still point to the old server. To redirect them to the new server, it is required to use the same FQDN and IP address for the new server as the old one.
```

Comment:
I would say change DHCP and DNS to assign the FQDN and IP address of the 4.3 server to the newly migrated 5.0 server.

Reply:
Sounds sensible.  I'll add this somehow as an additional sentence.